corehq/form_processor/backends/sql/dbaccessors.py

```diff
@@ -907,6 +907,24 @@ def get_reverse_indexed_cases(domain, case_ids, case_types=None, is_closed=None)
         _attach_prefetch_models(cases_by_id, indices, 'case_id', 'cached_indices')
         return cases
 
+    @staticmethod
+    def check_transaction_order_for_case(case_id):
+        """ Returns whether the order of transactions needs to be reconciled by client_date
+
+        True if the order is fine, False if the order is bad
+        """
+        if not case_id:
+            return False
+
+        from corehq.sql_db.util import get_db_alias_for_partitioned_doc
+        db = get_db_alias_for_partitioned_doc(case_id)
+        with connections[db].cursor() as cursor:
+            cursor.execute(
+                'SELECT compare_server_client_case_transaction_order(%s::text, %s)',
```

Comment:
is this cast to `text` necessary?

Reply:
It was weird, but it seemed to be. It was apparently looking for `compare_server_client_case_transaction_order("unknown", "integer")` otherwise. Seems to be working now, so maybe it was another issue and it can implicitly cast unknown to text